CODEOWNERS

```diff
@@ -1,2 +1,7 @@
 # Default owner
-*   @klaviyo/mobile-push-team
+*   @klaviyo/mobile-app-growth
+
+# Push related files
```

Comment:
I kept the native module to default to mobile app growth since they are a single file and I think app growth should have ownership on them. I tried co-owners but it's easy to clearly define ownership so figured we'll start with this and see organically what files push ends up having to make changes to and update as we go.